smalltalksrc/VMMaker/StackInterpreter.class.st

```diff
@@ -14989,7 +14990,7 @@ StackInterpreter >> subscript: array with: index storing: oopToStore format: fmt
 			 objectMemory storeShort16: index - 1 ofObject: array withValue: signedValueToStore.
 			 ^self].
 		 fmt = objectMemory sixtyFourBitIndexableFormat ifTrue:
-			[| unsigned64BitValueToStore |
+			[
```

Comment:
Why this change?

Reply:
Personally, I like temporary definitions close to their usages, but maybe you have a stronger reason :)

Reply:
Haha because Druid is not-too-much smart and makes the lookup of the variable directly in the outer context. Was easier to change the variable than fix it.

I also prefer the variable in the closest scope, but the rest of the code doesn't follow it. The IDE only adds temps to the method... So blah...

Reply:
https://github.com/Alamvic/druid/issues/108